processor/spanprocessor/span.go

```diff
@@ -121,3 +161,76 @@ func (sp *spanProcessor) nameSpan(span *tracepb.Span) {
 	}
 	span.Name = &tracepb.TruncatableString{Value: sb.String()}
 }
+
+func (sp *spanProcessor) processToAttributes(span *tracepb.Span) {
+	if span.Name == nil || span.Name.Value == "" {
+		// There is no span name to work on.
+		return
+	}
+
+	if sp.config.Rename.ToAttributes == nil {
+		// No rules to apply.
+		return
+	}
+
+	// Process rules one by one. Store results of processing in the span
+	// so that each subsequent rule works on the span name that is the output
+	// after processing the previous rule.
+	for _, rule := range sp.toAttributeRules {
+		re := rule.re
+		oldName := span.Name.Value
+
+		// Match the regular expression and extract matched subexpressions.
+		if submatches := re.FindStringSubmatch(oldName); submatches != nil {
```

Comment:
nit: I think it will be more readable (a few more lines though) by using `if submatches == nil { continue }`, personal preference...

Reply:
Done.